inc/page_cache/namespace.php

```diff
@@ -80,7 +80,7 @@ function get_unique_headers() : array {
 		$header_key = 'HTTP_' . str_replace( '-', '_', strtoupper( $header ) );
 		if ( isset( $_SERVER[ $header_key ] ) ) {
 			// Add header to batcache vary keys.
-			$unique_keys[ $header ] = $_SERVER[ $header_key ];
+			$unique_keys[ $header ] = wp_unslash( $_SERVER[ $header_key ] ); // phpcs:ignore HM.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
I'm pretty sure `wp_unslash()` won't be available yet as this runs super early during the bootstrap process. For long lines like this you can also put the `phpcs:ignore` rule on the line above.